recipe/meta.yaml

```
@@ -14,33 +14,39 @@ build:
     - pystache=pystache.commands.render:main
     - pystache-test=pystache.commands.test:main
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 1
 
 requirements:
   host:
-    - python >=3.6
-    - setuptools >=40.8.0
-    - wheel
+    - python >=3.8
+    - setuptools >=61
+    - setuptools-scm >=6.2
     - pip
   run:
-    - python >=3.6
+    - python >=3.8
+    # only required for py<310 but want to stay noarch: python
+    - importlib-metadata >=4.6
 
 test:
   imports:
     - pystache
   commands:
     - pip check
+    - pip show pystache
+    - "pip show {{ name }} | grep -Fx 'Version: {{ version }}'"
```

```suggestion
    - "pip show {{ name }} | grep -Fx 'Version: {{ version }}' || true"
```
This should work to raise an issue when no match is found, so we do not manually have to check the logs.

---

I'm a bit confused about this.  Won't `|| true` ensure that the command always succeeds (e.g. https://unix.stackexchange.com/questions/78408/which-is-more-idiomatic-in-a-bash-script-true-or).  I think that's not what we want.  I think the existing behavior was right.

---

I tested this locally with the existing `pystache` package:
```
$ pip show pystache | grep -Fx 'Version: 0.6.5' && echo success || echo failed 
failed
```
and it seems like it's doing what it's supposed to without the `|| true`.

---

yes, I got that wrong.